src/server/clients_info.rs

```diff
@@ -25,9 +21,13 @@ pub(crate) struct ClientBuffers {
     entities: Vec<Vec<Entity>>,
 }
 
+/// Stores meta-information about connected clients.
+#[derive(Default, Resource)]
+pub struct ClientsInfo(Vec<ClientInfo>);
+
 impl ClientsInfo {
     /// Returns a mutable iterator over clients information.
-    pub(super) fn iter_mut(&mut self) -> impl Iterator<Item = &mut ClientInfo> {
+    pub fn iter_mut(&mut self) -> impl Iterator<Item = &mut ClientInfo> {
```

Comment:
Is this change related to the change? If not, let's remove it to avoid conflicts with my rooms branch (I did the same change).

Reply:
Oh, I missed it. I'm not sure if we should export things like this to users.

Reply:
No... I will think about it.